src/checkers/__init__.py

```diff
@@ -120,7 +120,7 @@ async def _get_json(
                         return yaml.load(await response.read())
                     except ruamel.yaml.error.YAMLError as err:
                         raise CheckerQueryError("Failed to parse YAML") from err
-                return await response.json()
+                return await response.json(content_type=None)
```

Comment:
We'll need to wrap this in a try/except block to capture and re-raise json decode errors, because `response.json()` won't raise `aiohttp.client.ContentTypeError` anymore (which was captured in the outer try/except block).
```suggestion
                try:
                    return await response.json(content_type=None)
                except json.JSONDecodeError as err:
                    raise CheckerQueryError from err
```

And, ideally, please add a test case to verify that a known non-json url causes a `CheckerQueryError`.

Reply:
All done.